modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/prepare/Fragment.java

```diff
@@ -139,7 +133,7 @@ public Fragment detach() {
      * @param ctx Planner context.
      * @param mq Metadata query.
      */
-    Fragment map(MappingService mappingSrvc, PlanningContext ctx, RelMetadataQuery mq) throws FragmentMappingException {
+    Fragment map(MappingService mappingSrvc, MappingQueryContext ctx, RelMetadataQuery mq) throws FragmentMappingException {
         assert root.getCluster() == ctx.cluster() : "Fragment is detached [fragment=" + this + "]";
```

Comment:
currently this assertion will be always true, hence, probably, it's better to remove it. This allows us to remove `cluster` from a MappingQueryContext since this place is the only usage